opensquirrel/Circuit.py

```diff
@@ -1,42 +1,44 @@
-from parsing.GeneratedParsingCode import CQasm3Parser
-from parsing.GeneratedParsingCode import CQasm3Lexer
-from parsing.GeneratedParsingCode import CQasm3Visitor
-from src.TypeChecker import TypeChecker
-from src.McKayDecomposer import McKayDecomposer
-from src.SquirrelErrorHandler import SquirrelErrorHandler
-from src.SquirrelAST import SquirrelAST
-from src.SquirrelASTCreator import SquirrelASTCreator
-from src.TestInterpreter import TestInterpreter
-from src.Replacer import Replacer
-from src.Writer import Writer
 import antlr4
 
+from opensquirrel.McKayDecomposer import McKayDecomposer
+from opensquirrel.Replacer import Replacer
+from opensquirrel.SquirrelAST import SquirrelAST
+from opensquirrel.SquirrelASTCreator import SquirrelASTCreator
+from opensquirrel.SquirrelErrorHandler import SquirrelErrorHandler
+from opensquirrel.TestInterpreter import TestInterpreter
+from opensquirrel.TypeChecker import TypeChecker
+from opensquirrel.Writer import Writer
+from parsing.GeneratedParsingCode import CQasm3Lexer
+from parsing.GeneratedParsingCode import CQasm3Parser
+from test.TestGates import TEST_GATES as DefaultGates  # For the doctest.
```

Comment:
pretty sure this is now broken, since I moved test.TestGates to src.DefaultGates

Reply:
Yes, for some reason I didn't 'add' those files to the previous commit...